mods/cnc/maps/gdi05a/map.yaml

```diff
@@ -796,6 +801,7 @@ Rules:
 		-ConquestVictoryConditions:
 		MissionObjectives:
 			EarlyGameOver: true
+		EnemyWatcher:
```

Comment:
This is already added to the playeractor?
https://github.com/OpenRA/OpenRA/pull/7492/files#diff-9bee53d823b230f9649e14db8b380018R76